app/monitoringapi.go

```diff
@@ -133,19 +158,26 @@ func peersReady(ctx context.Context, peerIDs []peer.ID, tcpNode host.Host) error
 
 	var (
 		// Require quorum successes (excluding self). Formula from IBFT 2.0 paper https://arxiv.org/pdf/1909.10194.pdf
-		require = int(math.Ceil(float64(len(peerIDs)*2)/3)) - 1
-		actual  int
+		require  = int(math.Ceil(float64(len(peerIDs)*2)/3)) - 1
+		actual   int
+		errCount int
 	)
 	for {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
 		case res := <-results:
 			if res.Error != nil {
-				continue
+				errCount++
```

Comment:
added error counter to return error if are unable to ping quorum number of peers